src/clipmenud.c

```diff
@@ -384,6 +385,14 @@ int main(void) {
     int evt_base;
 
     cfg = setup("clipmenud");
+
+    _drop_(close) int session_fd =
```

Comment:
Probably also O_CLOEXEC?

Reply:
Done. But does clipmenud do any `exec`? Also what about `content_dir_fd` and `snip_fd`, shouldn't they be `O_CLOEXEC`ed too if it's just out of precaution/principle?

Reply:
`O_CLOEXEC` is more important on `flock()`ed files than regular files because `flock()` maintains locking semantics across process images. It would be cleaner elsewhere, too, but less essential.

We do `fork()`/`exec()` on `run_clipserve()`.